### ISSUE TITLE
required_pull_request_reviews in github_branch_protection is missing the list of authorized bypass 

### ISSUE THREAD

**Issue by jdfischer-cedreo** [integrations/terraform-provider-github#1080]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Terraform v0.12.31
provider.github v4.20.0

### Affected Resource(s)
- github_branch_protection 

### Terraform Configuration Files
```hcl
resource "github_branch_protection" "test-branch-protection-master" {
  repository_id = github_repository.test-repository.node_id
  pattern = "master**"
  required_pull_request_reviews {
    bypass_pull_request = [
      data.github_user.devtools.node_id,
    ]
  }
}
```

### Expected Behavior
A field to specify the list of user that can bypass pull request. The label in github interface is "Allow specified actors to bypass pull request requirements".

### Actual Behavior
No option to do this.

**Comment by btkostner** [comment]
This was added and released in 4.21.0 https://github.com/integrations/terraform-provider-github/pull/1030

**Comment by kfcampbell** [comment]
CLosing as #1030 has been merged.
